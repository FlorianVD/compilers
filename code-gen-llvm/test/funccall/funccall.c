void do_print() { print_s("Test"); }

int main() {
    print(1);
    print_f(1.0);
    print8(1, 2, 3, 4, 5, 6, 7, 8);

    print_s("Hello, world!");
    do_print();

    print(read());
    print_f(read_f());

    return 0;
}
