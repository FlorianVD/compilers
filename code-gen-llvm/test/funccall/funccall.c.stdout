1
1.00000
Hello, world!
Test
> 10
> 20.5000
